loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -115,7 +115,7 @@ export class GLTFLoader implements IGLTFLoader {
     private _gltf: IGLTF;
     private _bin: Nullable<IDataBuffer>;
     private _babylonScene: Scene;
-    private _rootBabylonMesh: Mesh;
+    private _rootBabylonMesh?: Mesh;
```

Comment:
Based on the other comments below. I would recommend changing this to:
```suggestion
    private _rootBabylonMesh: Nullable<Mesh> = null;
```

Reply:
I applied your suggestion!